File: src/G1_rust/PYA104.rs
use std::io;
use std::f64::consts::PI;

fn main(){
    let mut input = String::new();
    io::stdin().read_line(&mut input).unwrap();
    let radius: f64 = input.trim().parse().unwrap();
    let perimeter: f64 = 2.0*PI*radius;
    let area = (PI*radius).powi(2);

    println!("Radius = {:.2}",radius);
    println!("Perimeter = {:.2}",perimeter);
    println!("Area = {:.2}",area);

}

File: src/G1_py/PYA104.py
from math import pi,pow

radius=float(input())

perimeter = 2*pi*radius
area = pi*pow(radius,2)


print("Radius = {:.2f}".format(radius))
print("Perimeter = {:.2f}".format(perimeter))
print("Area = {:.2f}".format(area))
